tests/hpc/product_migration.pm

```diff
@@ -21,12 +21,15 @@ use registration 'add_suseconnect_product';
 sub run {
     my $self            = shift;
     my $suseconnect_str = ' -e testing@suse.com -r ';
+    my $version         = get_required_var('VERSION');
+    ## replace SP-X with SP.X as this form is expected by SUSEConnect
+    $version =~ tr/-/./;
     $self->select_serial_terminal;
 
     script_run('ls -la /etc/products.d/');
     my $out = script_output('SUSEConnect -s', 30, proceed_on_failure => 1);
     assert_script_run('SUSEConnect --cleanup', 200) if $out =~ /Error: Invalid system credentials/s;
-    add_suseconnect_product('SLES', '12.4', get_var('ARCH'), $suseconnect_str . get_required_var('SCC_REGCODE'));
+    add_suseconnect_product('SLES', '12.5', get_required_var('ARCH'), $suseconnect_str . get_required_var('SCC_REGCODE'));
```

Comment:
```suggestion
    add_suseconnect_product('SLES', $version, get_required_var('ARCH'), $suseconnect_str . get_required_var('SCC_REGCODE'));
```

Reply:
@jlausuch sure ;-) sorry I messed up the regexp so I forced-pushed you hardcoded value as it was before.